spec/quotas/quotas.mdown

```diff
@@ -0,0 +1,147 @@
+
+This specification defines a data model for handling mail quotas over JMAP, allowing you to read and explain quota information.
+
+This specification does not address quota administration, which should be handled by other means.
+
+## Notational conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in BCP
+14 [@!RFC2119] [@!RFC8174] when, and only when, they appear in all
+capitals, as shown here.
+
+Type signatures, examples and property descriptions in this document follow the conventions established in section 1.1 of [@!RFC8620]. Data types defined in the core specification are also used in this document.
+
+Servers MUST support all properties specified for the new data types defined in this document.
+
+## Terminology
+
+### Quota
+
+A quota is a numeric upper limit that the server is enforcing. Quotas are applied to JMAP Mailbox objects.
+
+## Addition to the capabilities object
+
+The capabilities object is returned as part of the JMAP Session object; see [@!RFC8620], section 2.
+
+This document defines one additional capability URI.
+
+### urn::ietf::params::jmap::quota
+
+This represents support for the Quota data type and associated API methods. Servers supporting this specification MUST add this property to the capabilities object.
+
+# Quota definition
+
+The quota is an object that displays the limit set to a mailbox usage as well as the current usage in regard to that limit.
+
+## The Scope Data Type
+
+The **Scope** is a `String` from an enumeration defined list of values, handled by the server.
+
+It explains the entities this value applies to. Some custom specifications might provide some additional values. If the client does not specify custom scope specifications in the "using" parameter of the request, the server should respond the JSON value `null`, instead of answering a scope value that the client does not support. Standard values are:
+* `account`: Applies for this account
+* `domain`: All users of this domain share this part of the quota
+* `global`: All users of this mail server share this part of the quota
+
+## The Quota Object
+
+The quota object MUST contain the following fields:
+
+* **id**: `Id` The unique identifier for this object. It should respect the JMAP ID datatype defined in section 1.2 of [@!RFC8620]
+
+* **type**: `String` The type of quota. It should either be "messageCount" or "messageStorageSize". There is two ways we can define a quota: either on a number of emails, or in the amount of bytes.
+
+* **used**: "UnsignedInt" The current usage of the mailbox. Computation of this value is handled by the server.
+
+* **usedScope**: `Scope` The `Scope` of the current mailbox usage.
```

Comment:
what does "current" mean here?

Reply:
The usage we have at this moment? Or you think it's a bit clumsy and can find better here?